diff --git a/node_modules/astro-imagetools/integration/index.js b/node_modules/astro-imagetools/integration/index.js
index b64f6ed..3a0b7c9 100755
--- a/node_modules/astro-imagetools/integration/index.js
+++ b/node_modules/astro-imagetools/integration/index.js
@@ -43,5 +43,6 @@ export default {
         },
       });
     },
+    "astro:config:done": vitePluginAstroImageTools.closeBundle,
   },
 };
\ No newline at end of file
