diff --git a/node_modules/astro-imagetools/api/utils/getProcessedImage.js b/node_modules/astro-imagetools/api/utils/getProcessedImage.js
index 61b5d08..acc9e86 100755
--- a/node_modules/astro-imagetools/api/utils/getProcessedImage.js
+++ b/node_modules/astro-imagetools/api/utils/getProcessedImage.js
@@ -2,6 +2,7 @@
 import fs from "node:fs";
 import crypto from "node:crypto";
 import { join, basename, extname, relative, resolve } from "node:path";
+import fileType from 'file-type'
 import {
   cwd,
   sharp,
@@ -15,8 +16,9 @@ const { getImageDetails } = await (sharp
   ? import("./imagetools.js")
   : import("./codecs.js"));
 
-// @ts-ignore
-const { fileTypeFromBuffer } = await import("file-type");
+
+// const { fileTypeFromBuffer } = await import("file-type");
+const {fileTypeFromBuffer} = fileType
 
 const throwErrorIfUnsupported = (src, ext) => {
   if (!ext && typeof ext !== "string") {
diff --git a/node_modules/astro-imagetools/integration/index.js b/node_modules/astro-imagetools/integration/index.js
index b64f6ed..f948ae2 100755
--- a/node_modules/astro-imagetools/integration/index.js
+++ b/node_modules/astro-imagetools/integration/index.js
@@ -43,5 +43,6 @@ export default {
         },
       });
     },
+    "astro:build:done": vitePluginAstroImageTools.closeBundle,
   },
 };
